idaes/models/costing/tests/test_SSLW_costing.py

```diff
@@ -851,13 +852,46 @@ def test_pfr(self, model):
 
     def test_pressure_changer(self, model):
         # Add examples of supported unit models and add costing
-        model.fs.unit = PressureChanger(default={"property_package": model.fs.pparams})
+        model.fs.unit = PressureChanger(
+            default={
+                "property_package": model.fs.pparams,
+                "thermodynamic_assumption": ThermodynamicAssumption.isentropic,
+            }
+        )
         model.fs.unit.costing = UnitModelCostingBlock(
             default={"flowsheet_costing_block": model.fs.costing}
         )
         assert model.fs.unit.costing.drive_factor.value == 1
         assert model.fs.unit.costing.material_factor.value == 2.5
 
+    def test_isothermal_compressor(self, model):
+        # Test exception for non-supported compressor flags
+        model.fs.unit = PressureChanger(
+            default={
+                "property_package": model.fs.pparams,
+                "compressor": True,
+                "thermodynamic_assumption": ThermodynamicAssumption.isothermal,
+            }
+        )
+        with pytest.raises(ValueError):
```

Comment:
This is only a suggestion for a possible enhancement: to make this test more specific, you can specify a substring or pattern that the exception message is expected to contain using the `match` kwarg: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-raises.

Reply:
This is very useful, thank you for sharing!